tests/container_hardening/test_container_hardening.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import logging
+from tests.common.helpers.assertions import pytest_assert
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+logger = logging.getLogger(__name__)
+
+
+def test_bgp_dev(duthost):
+    """
+    Test bgp container has no access to /dev/vda* or /dev/sda*
```

Comment:
Is this test related to the --privileged flag removal or net=host?
I think we should mention this and also change the name of the test to accommodate it.
Also - in the future, I think we should make this test generic for any container that wants to test their solution.

Reply:
@Yarden-Z it's related to test --privileged flag, I modified the test to make it generic.

Reply:
It's a good improvement.
I think that in the future we can make this test even more generic.
i.e - maintain a list of containers that do not have the --privileged flag. (define this list in some global or maintain it in the section of container hardening)
Then - for each container (name) in this list, iterate on top of them and execute this test. That way - we won't need the if case and we'll be able to scale this test for each new container.

Reply:
addressed your comment, thanks